copyables/entrypoint.sh

```
@@ -155,15 +155,15 @@ if [ ! -f $CONFIG ] || [ ! -s $CONFIG ]; then
   export PASSWORD='**'
 
   # handle VPNCMD_* commands right before setting admin passwords
-  if [[ $VPNCMD_SERVER ]]; then
-    while IFS=";" read -ra CMD; do
-      vpncmd_server $CMD
+  if [[ "$VPNCMD_SERVER" ]]; then
+    while IFS=";" read -r -a CMD; do
+      vpncmd_server "$CMD"
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)